src/ImageCard.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class ImageCard extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      showResults: false
```

Comment:
You can simplify the rendering logic if you set `this.state.showResults` initially to be `content.length <= 5`.